src/instrument/initialize_bs_tools.py

```diff
@@ -1,126 +1,77 @@
 """
-initialize the bluesky framework
+Initialize bluesky tools
 """
 
 __all__ = """
     RE  cat  sd  bec  peaks
-    bp  bps  bpp
-    summarize_plan
-    np
     oregistry
     """.split()
 
 import logging
 
-logger = logging.getLogger(__name__)
-
-logger.info(__file__)
-
-import pathlib
-import sys
-
-sys.path.append(str(pathlib.Path(__file__).absolute().parent.parent.parent))
-
-import warnings
-
+# convenience imports
 import databroker
-import numpy as np
 import ophyd
-from IPython import get_ipython
-from ophyd.signal import EpicsSignalBase
-
-# convenience imports
-import bluesky.plan_stubs as bps
-import bluesky.plans as bp
-import bluesky.preprocessors as bpp
-from bluesky import RunEngine, SupplementalData
+from bluesky import SupplementalData
 from bluesky.callbacks.best_effort import BestEffortCallback
-from bluesky.magics import BlueskyMagics
-from bluesky.simulators import summarize_plan
-from bluesky.utils import PersistentDict, ProgressBarManager, ts_msg_hook
+from ophyd.signal import EpicsSignalBase
 from ophydregistry import Registry
 
-from .. import iconfig
+from .utils.catalog import load_catalog
+from .utils.config_utils import iconfig
+from .utils.metadata import MD_PATH
+from .utils.run_engine import run_engine
 
+# TODO: This is not inside init. Some aspects do not work with qserver.
 
-def get_md_path():
-    path = iconfig.get("RUNENGINE_MD_PATH")
-    if path is None:
-        path = pathlib.Path.home() / "Bluesky_RunEngine_md"
-    else:
-        path = pathlib.Path(path)
-    logger.info("RunEngine metadata saved in directory: %s", str(path))
-    return str(path)
+logger = logging.getLogger(__name__)
+logger.info(__file__)
```

Comment:
again, local (relative) imports after `logger.info(__file__)`

Reply:
I'm unsure if I understand why we would want the logging happening before the relative imports. Almost all the files have their own logger so when they start it will be logged. As well when an issue happens the terminal will point to the line where the issue happens

Reply:
Preserves the order of which one came first.  Helps to identify after what point a problem started. When `logger.info()` is called after a buggy relative import, it is not possible to see this order clearly in the log files.

This is a surveillance.  Beam line users often report a problem occurred but give no detail about the occurrence.  Often, they miss key detail.  This type of surveillance is useful to preserve what actually happened.